theme/management/commands/migrate_quota_to_minio.py

```diff
@@ -0,0 +1,28 @@
+import subprocess
```

Comment:
This migration management command needs to be run before the django migrations. I was thinking i would just copy this file into the cotainer and run it before doing a deploy.